### PR TITLE
Clarify some confusions about printing of real balls

### PR DESCRIPTION
**Pull request by albinahlback**
Adds documentation to clafiry the printing of real balls leading to issues like #798 and #850.

### CODE REVIEW

File: docs/src/arb.md [hunk 1]
```diff
@@ -170,6 +170,34 @@ d = midpoint(b)
...
+[+/- 3.01]
+
+julia> midpoint(c)
+1.000000000
+
+julia> radius(c)
+[2.000000004 +/- 2.75e-10]
+```
+
+Note that Nemo does not print `c` as `[1 +/- 2]`. This is because the midpoint
```
Comment (albinahlback) [inline docs/src/arb.md:195]: Maybe instead write "One part of this is because..."

File: docs/src/arb.md [hunk 2]
```diff
@@ -173,30 +173,42 @@ f = accuracy_bits(a)
...
-Note that Nemo does not print `c` as `[1 +/- 2]`. This is because the midpoint
-does not have a greater exponent than the radius in its scientific notation.
+The first reason that `c` does not get printed as `[1 +/- 2]` is because the
+midpoint does not have a greater exponent than the radius in its scientific
+notation. By similair reasons we also have that `y` does not get printed as
+`[12 +/- 2]`.
 
-We also have that `radius(c)` is not strictly equal to $2$, which occurs
-because real balls keeps track of rounding errors.
+The second reason is that we an additional error term after our addition. As we
```
Comment (thofma) [inline docs/src/arb.md:208]:
```suggestion
The second reason is that we get an additional error term after our addition. As we
```
?

**Review by thofma — APPROVED** [review]

File: docs/src/arb.md (hunk 2 above)
Comment (albinahlback) [reply docs/src/arb.md]: Good, thank you.

**Comment by fingolfin** [comment]
@thofma @wbhart can this be merged?

**Pull request merged**